registry/consul/instance.go

```diff
@@ -0,0 +1,18 @@
+package consul
+
+import "github.com/moderntv/cadre/registry"
+
+var _ registry.Instance = &instance{}
```

Comment:
Variable after type definition

Reply:
Again, I disagree based on our existing code. All interface assertions like these are made at the start of the file where the structure is defined. I would much rather see these assertions as the first thing rather than to found them between structure definition and some of its methods.